Currently we print the raw UUID which isn't very useful in ascii format.

Signed-off-by: Cole Robinson <crobinso@redhat.com>
---
 src/qemu_driver.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/qemu_driver.c b/src/qemu_driver.c
index 40f5790..9bd7d03 100644
--- a/src/qemu_driver.c
+++ b/src/qemu_driver.c
@@ -2043,7 +2043,7 @@ static virDomainPtr qemudDomainLookupByUUID(virConnectPtr conn,
         char uuidstr[VIR_UUID_STRING_BUFLEN];
         virUUIDFormat(uuid, uuidstr);
         qemudReportError(conn, NULL, NULL, VIR_ERR_NO_DOMAIN,
-                         _("no domain with matching uuid '%s'"), uuid);
+                         _("no domain with matching uuid '%s'"), uuidstr);
         goto cleanup;
     }
 

